polyfactory/factories/sqlalchemy_factory.py

```diff
@@ -148,7 +151,7 @@ def get_model_fields(cls) -> list[FieldMeta]:
                 name=name,
                 random=cls.__random__,
             )
-            for name, column in table.columns.items()
+            for name, column in table.mapper.columns.items()
```

Comment:
Just out of curiosity, what is the difference between `table.columns` and `table.mapper.columns`?

Reply:
Ah I accidentally added this when trying to see if it was different. I'll revert.